lewis_emulators/knr1050/interfaces/stream_interface.py

```diff
@@ -0,0 +1,122 @@
+from lewis.adapters.stream import StreamInterface
+
+from lewis.core.logging import has_log
+from lewis_emulators.utils.replies import conditional_reply
+from lewis_emulators.utils.command_builder import CmdBuilder
+
+if_connected = conditional_reply('connected')
+
+
+@has_log
```

Comment:
Remove the `has_log` decorator and any logging from the emulator when you create the pull request.